src/reducers/arcReducer.ts

```diff
@@ -298,7 +298,7 @@ const arcReducer = (state = initialState, action: any) => {
 
   switch (action.type) {
     case ActionTypes.ARC_LOAD_CACHED_STATE_FULFILLED: {
-      return payload;
+      return update(state, { $merge: payload});
```

Comment:
Not sure about this change.  The reason is that the payload (i.e. the S3 initial state) does not have an `accounts` key, and this leads to an error in https://github.com/daostack/alchemy/blob/dev/src/layouts/HeaderContainer.tsx#L42

Reply:
hmm, it should (and does) have an accounts key, when/where are you seeing this issue?

Reply:
yeah, this was becuase of my confusion about the location of the initial state, cf #201, I undid this change in a later commit